auditwheel/wheel_abi.py

```diff
@@ -112,7 +113,46 @@ def analyze_wheel_abi(wheel_fn: str):
     log.debug('external reference info')
     log.debug(json.dumps(external_refs, indent=4))
 
+    # get versioned symbols from external_refs
+    external_ref_no_linux = []
+    for e in external_refs.values():  # go through all policies
+        if e['priority'] == 0:
+            continue    # since linux tag (priority 0) has no white-list, do not analyze it
+        for libname, realpath in e['libs'].items():  # go through all referenced libs, retrieving their name and realpath
+            if realpath is None:
+                continue  # we can't analyze the lib if we don't know where it's located
+            if (realpath, libname) not in external_ref_no_linux:
+                external_ref_no_linux.append((realpath, libname))
+    external_ref_no_linux = dict(external_ref_no_linux)  # create a dict using the realpath as a key, libname as a value
+    external_refs_symbol_policy = {}  # create a dict using libname as a key, versioned symbols as a value
```

Comment:
A comment like this could more concisely be written: `# {libname: versioned_symbols}`

But it's still a bit hard to follow what's going on. Can we expand a bit more what versioned symbols are?

```python
external_refs_symbol_policy = {}  # {libname: {???: set([???]) }}
```

It might also be useful to give some example values alongside the prototype:

```python
# {'libc.so.6': {???: set([???]) }}
```